falcon/response.py

```diff
@@ -114,6 +132,12 @@ def __init__(self):
         self.stream = None
         self.stream_len = None
 
+        if self.context_type is None:
+            # PERF(kgriffs): The literal syntax is more efficient than dict().
+            self.context = {}
+        else:
+            self.context = self.context_type()
+
```

Comment:
I personally am a fan of using ternary operators, but I've met so many people who are against it, so now I avoid them. +1 for keeping it as is since it is easier to read.